spine/error.go

```diff
@@ -24,3 +28,10 @@ func NewErrorTypeFromResult(result *model.ResultDataType) *ErrorType {
 		Description: *result.Description,
 	}
 }
+
+func (e *ErrorType) Error() string {
```

Comment:
`String() string` wäre besser, da man dann das Objekt z.B. direkt in `fmt.Sprintf` verwenden kann.
(siehe auch [https://go.dev/tour/methods/17](https://go.dev/tour/methods/17)